ethereum/spec/src/test/java/tech/pegasys/teku/spec/config/SpecConfigLoaderTest.java

```diff
@@ -46,16 +44,16 @@ public void shouldLoadAllKnownNetworks(final Eth2Network network) throws Excepti
     assertAllFieldsSet(config, SpecConfigElectra.class);
   }
 
-  /**
+  /***
```

Comment:
What's up with the 3 `*`? :)

Reply:
because spotless complained loudly, i dont know.

Reply:
will see if its less unhappy now, but it was adamant.